Form/Type/CustomEntityType.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Form\Type;
+
+use Pim\Bundle\EnrichBundle\Form\Subscriber\DisableFieldSubscriber;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Rémy Bétus <remy.betus@akeneo.com>
+ */
+class CustomEntityType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('code')
+            ->addEventSubscriber(new DisableFieldSubscriber('code'));
+    }
+
+    public function getName()
+    {
+        return 'pim_enrich_custom_entity';
```

Comment:
pim_custom_entity_reference_data ?

pim_enrich should not appear here I think. The form is in pim_custom_entity